spec/cocaine/command_line/runners/backticks_runner_spec.rb

```diff
@@ -4,12 +4,14 @@
   if Cocaine::CommandLine::BackticksRunner.supported?
     it_behaves_like 'a command that does not block'
 
-    it 'runs the command given' do
-      subject.call("echo hello").should == "hello\n"
+    it 'runs the command given and captures the output in an Output' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.